addons/explosives/XEH_postInit.sqf

```diff
@@ -36,16 +36,31 @@ GVAR(Setup) = objNull;
 GVAR(pfeh_running) = false;
 GVAR(CurrentSpeedDial) = 0;
 
-// Properly angle preplaced bottom-attack SLAMs
-{
-    if (local _x) then {
-        switch (typeOf _x) do {
-            case ("ACE_SLAMDirectionalMine_Magnetic_Ammo"): {
-                [_x, getDir _x, 90] call FUNC(setPosition);
-            };
-        };
-    };
-} forEach allMines;
+// In case we are a JIP client, ask the server for orientation of any previously
+// placed mine.
+if (isServer) then {
+    ["clientRequestsOrientation", {
+        params ["_client"];
+        TRACE_1("clientRequestsOrientations received:",_client);
+        TRACE_2("serverSendsOrientations sent:",GVAR(explosivesOrientations));
+        ["serverSendsOrientations", _client, GVAR(explosivesOrientations)] call EFUNC(common,targetEvent);
+    }] call EFUNC(common,addEventHandler);
+} else {
+    ["serverSendsOrientations", {
+        params ["_explosivesOrientations"];
+        TRACE_1("serverSendsOrientations received:",_explosivesOrientations);
+        {
+            params ["_explosive","_direction","_pitch"];
+            [_explosive, _direction, _pitch] call FUNC(setPosition);
+        } forEach _explosivesOrientations;
+    }] call EFUNC(common,addEventHandler);
+
+    //  Create a logic to get the client ID
+    private _logic = createGroup sideLogic createUnit ["Logic", [0,0,0], [], 0, "NONE"];
```

Comment:
I'm still in the market for a better way of doing this. If we find it we should probably use it in ACE_Markers too